web/modules/custom/joinup_user/joinup_user.behat.inc

```diff
@@ -121,4 +124,25 @@ class JoinupUserSubContext extends DrupalSubContextBase implements DrupalSubCont
     $this->assertSession()->elementExists('css', "[src*='$filename.$extension']");
   }
 
+  /**
+   * Replaces human readable values with their real counterparts for users.
+   *
+   * @param \Drupal\DrupalExtension\Hook\Scope\BeforeUserCreateScope $scope
+   *   An object containing the entity properties and fields that are to be used
+   *   for creating the user as properties on the object.
+   *
+   * @throws \Exception
+   *   Thrown when some of the values are not one of the expected ones.
+   *
+   * @BeforeUserCreate
+   */
+  public static function massageFieldsBeforeUserCreate(BeforeUserCreateScope $scope) {
```

Comment:
It took me a while but I found where we convert the other field labels for the user.
It's in `\Drupal\joinup\Traits\UserTrait::createUser()`, which is invoked from `\Drupal\joinup\Context\DrupalContext::createUsers()`. We can move this there so all the related code is together.

Reply:
I am reverting actually. We already have tests failing because of this.

Reply:
Yes we should refactor the whole thing as we shouldn't put traits in traits. Ok for now.